.github/workflows/sycl-linux-build.yml

```diff
@@ -109,6 +109,9 @@ jobs:
   build:
     name: Build + LIT
     runs-on: [Linux, build]
+    defaults:
+      run:
+         shell: bash
```

Comment:
Is this necessary for `sycl-linux-build.yml`?

Reply:
i forget so let me try to remove it :P

Reply:
removed in https://github.com/intel/llvm/pull/16652/commits/3f12ef24b9f908a9f85984be82bc75be76bbdef0 thx